### PR TITLE
Add SNO with fips profile for recert test 

### PR DESCRIPTION
**Pull request by wangke19**
Based on the epic https://issues.redhat.com/browse/MGMT-15533, we need to test recert tool on SNO, need to cover  more test profile.

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.14-e2e-aws-ovn-single-node-recert-serial

**Comment by wangke19** [comment]
/pj-rehearse pull-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aws-sno-recert-serial

**Comment by wangke19** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-qe

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aws-sno

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aws-sno

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-installer-rehearse-aws-sno-recert

**Comment by wangke19** [comment]
/pj-rehearse  pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-installer-rehearse-aws-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-installer-rehearse-aws-sno-fips-recert

**Comment by wangke19** [comment]
pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-installer-rehearse-aws-sno-fips-recert

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-recert-main-installer-rehearse-aws-sno-fips-recert

**Comment by wangke19** [comment]
@mresvanis, @jianlinliu   I added one new SNO profile to recert test, please help to review when you are free.
/cc @xingxingxia

File: ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml [hunk 1]
```diff
@@ -126,6 +126,17 @@ tests:
     - chain: openshift-e2e-test-qe
     workflow: openshift-e2e-aws-single-node-recert
   timeout: 8h0m0s
+- as: installer-rehearse-aws-sno-fips-recert
```
Comment (jianlinliu) [inline ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:129]:
```suggestion
- as: aws-ipi-sno-fips-recert
```

**Comment by mresvanis** [comment]
Thank you @wangke19! LGTM

**Comment by wangke19** [comment]
@mresvanis Please help approve the PR. Thanks.

File: ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml [hunk 2]
```diff
@@ -126,6 +126,17 @@ tests:
...
+  cron: '@weekly'
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FIPS_ENABLED: "true"
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-aws-ipi-sno-recert
+  timeout: 8h0m0s
```
[the remark this answers is not in the document]
Comment (xingxingxia) [reply ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:139]: MGMT-16060 intends to add openshift e2e test. This PR not yet includes e2e. Will separate PR follow up to include e2e?

[the remark this answers is not in the document]
Comment (xingxingxia) [reply ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:139]: And follow-up PR might need to put e2e tests under QE's https://github.com/openshift/release/tree/master/ci-operator/config/openshift/openshift-tests-private

**Pull request merged**